cloudgunnetworking/GenerateVPC.py

```
@@ -465,6 +495,42 @@ def _add_s3_vpc_endpoints(self, route_table_ids, vpc_id):
                 VpcId=vpc_id
         ))
 
+    def _create_nacl(self, tier, vpc_id):
+        return self.add_resource(ec2.NetworkAcl(
+            self.service_name_vpc + self.env_name_vpc + tier + "NACL",
+            VpcId=Ref(vpc_id),
+            Tags=self.id_tags + Tags(
+                Name=self.service + "-" + self.environment + "-" + tier,
+                Tier=tier
+            )
+        ))
+
+    def _create_nacl_entry(self, tier, egress, rule_num, cidr_str, port_range_str, protocol, action, acl):
+        port_range = port_range_str.split('-')
+        if len(port_range) == 1:
```

If len(port_range) would be > 1, the split() function on line 509 would already throw an exception. If it's >2, we should throw an exception (Somebody has configured port range such as 22-80-1024 or something as stupid).